src/imputation/MoR.py

```diff
@@ -288,33 +289,29 @@ def group_calc_link(group, target_vars, config):
         link_vars ([string]): List of the linked variables.
         config (Dict): Confuration settings
     """
-    valid_group_size = True
+    threshold_num = get_threshold_value(config)
```

Comment:
In function group_calc_link, please add type hints and what it Returns

Reply:
done :-)